src/controllers/jobsController.js

```diff
@@ -11,9 +11,13 @@ const getJobs = async (req, res) => {
 
     // Build query object
     const query = {};
-    if (search) query.title = { $regex: search, $options: 'i' }; // Case-insensitive search
+    if (search) {
+      query.$or = [
+      {title: { $regex: search, $options: 'i' }}, // Case-insensitive search
+      { description: { $regex: search, $options: 'i' } }
+      ];
```

Comment:
Is there a particular reason why you are using regex for searching? Regex can be quite slow when searching on multiple fields. I recommend adding index and use text search:
```suggestion
 // Add index
Job.collection.createIndex({ title: 'text', description: 'text' });

// Use text search
if (search) {
  query.$text = { $search: search };
}
```